interface/batchcom/emailnotification.php

```diff
@@ -73,45 +96,57 @@
 ?>
 <html>
 <head>
-<link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
-<link rel="stylesheet" href="batchcom.css" type="text/css">
-
-
+    <?php Header::setupHeader(['datetime-picker']); ?>
+    <title><?php xl("Email Notification"); ?></title>
 </head>
 <body class="body_top">
-<span class="title"><?php include_once("batch_navigation.php");?></span>
-<span class="title"><?php xl('Email Notification','e')?></span>
-<br><br>
-<!-- for the popup date selector -->
-<div id="overDiv" style="position:absolute; visibility:hidden; z-index:1000;"></div>
-<FORM name="select_form" METHOD=POST ACTION="">
-<input type="Hidden" name="type" value="<?php echo $type;?>">
-<input type="Hidden" name="notification_id" value="<?php echo $notification_id;?>">
-<div class="text">
-    <div class="main_box">
-        <?php
-        if ($form_err) {
-            echo ("The following errors occurred<br>$form_err<br><br>");
-        }
-        if ($sql_msg) {
-            echo ("$sql_msg<br><br>");
-        }
-        ?>
-        <?php xl('Email Sender','e')?> :
-        <INPUT TYPE="text" NAME="email_sender" size="40" value="<?php echo $email_sender?>">
-        <br>
-        <?php xl('Email Subject','e')?> :
-        <INPUT TYPE="text" NAME="email_subject" size="60" value="<?php echo $email_subject?>">
-        <br>
-
-        <?php xl('Name of Provider','e')?> :
-        <INPUT TYPE="text" NAME="provider_name" size="40" value="<?php echo $provider_name?>">
-        <br>
-        <?php xl('SMS Text, Usable Tag: ***NAME***, ***PROVIDER***, ***DATE***, ***STARTTIME***, ***ENDTIME***<br> i.e. Dear ***NAME***','e')?>
-        <br>
-        <TEXTAREA NAME="message" ROWS="8" COLS="35"><?php echo $message?></TEXTAREA>
-        <br><br>
-        <INPUT TYPE="submit" name="form_action" value="Save">
-    </div>
-</div>
-</FORM>
+    <?php include_once("batch_navigation.php");?>
+    <header class="text-center">
+        <h1>
+            <?php xl('Batch Communication Tool','e'); ?>
+            <small><?php xl('Email Notification','e')?></small>
+        </h1>
+    </header>
+    <main class="container">
+        <?php if ($form_err) { echo "<div class=\"alert alert-danger\">".xl("The following errors occurred").": $form_err</div>"; } ?>
+        <?php if ($sql_msg) { echo "<div class=\"alert alert-info\">".xl("The following errors occurred").": $sql_msg</div>"; } ?>
+        <form name="select_form" method="post" action="">
+            <input type="Hidden" name="type" value="Email">
+            <input type="Hidden" name="notification_id" value="<?php echo $notification_id;?>">
+            <div class="row">
```

Comment:
In below code:
1. change to xlt
2. use attr()
3. translate the placeholders (use xla())